testdata/sample_ips.log

```diff
@@ -0,0 +1,337 @@
+135.181.19.106:16125 is connected to itself.
+89.58.53.56:16125 is connected to itself.
+5.9.7.62:8233 is connected to itself.
+64.32.48.35:16125 is connected to itself.
+39.97.172.77:8233 is connected to itself.
+5.161.64.105:16125 is connected to itself.
+188.80.237.253:16125 is connected to itself.
+217.182.11.209:16125 is connected to itself.
+181.214.147.208:16125 is connected to itself.
+146.59.209.100:16125 is connected to itself.
+37.221.193.127:16125 is connected to itself.
+213.32.12.27:16125 is connected to itself.
+192.99.0.47:16125 is connected to itself.
+162.19.206.111:16125 is connected to itself.
+66.94.120.42:16125 is connected to itself.
+185.16.61.122:16125 is connected to itself.
+185.78.165.203:16125 is connected to itself.
+37.120.179.110:16125 is connected to itself.
+57.128.154.157:16125 is connected to itself.
+142.132.144.56:16125 is connected to itself.
+57.128.152.20:16125 is connected to itself.
+Generating initial network state and its statistics... 
+Statistics for the initial network:
+----------------------------------------
+Nodes count: 2472
+
+Degree measures:
+Average: 347.84021035598704
+Median: 362
+Min: 4, max: 468, delta: 464
+
+Betweenness measures:
+Average: 0.00020269540490710155
+Median: 0.00021315968556486295
+Min: 0.000000011402470586985047, max: 0.0005637115400558378, delta: 0.0005637001375852508
+
+Closeness measures:
+Average: 2.00092942092042
+Median: 1.9982984574223353
+Min: 1.996533523649682, max: 2.995343149165006, delta: 0.9988096255153243
+
+Eigenvector measures:
+Average: 1.0000000000000016
+Median: 1.0408706699768735
+Min: 0.009224464066643153, max: 1.338099762915545, delta: 1.3288752988489019
+----------------------------------------
+
+Generated initial state and statistics in 16 s
+IPS detected no islands
+IPS detected no fragmentation possibility even when top nodes would be disconnected
+All IPS computations done in 18 s from IPS start
+Statistics for the final network:
+----------------------------------------
+Nodes count: 2472
+
+Degree measures:
+Average: 349.40291262135923
+Median: 364
+Min: 6, max: 464, delta: 458
+
+Betweenness measures:
+Average: 0.00020263542604265325
+Median: 0.0002134924354306151
+Min: 0.000000011402470586985047, max: 0.0005637115400558378, delta: 0.0005637001375852508
+
+Closeness measures:
+Average: 2.0007482066942184
+Median: 1.9995108535613246
+Min: 1.996533523649682, max: 2.995343149165006, delta: 0.9988096255153243
+
+Eigenvector measures:
+Average: 1.0000000000000033
+Median: 1.040660344464556
+Min: 0.017939250553959517, max: 1.319354521636466, delta: 1.3014152710825064
+----------------------------------------
+
+Comparing if network parameters got changed on plus or minus:
+Deltas for given statistics pair:
+----------------------------------------
+Nodes count: 0 (0.000%)
+
+Degree measures:
+Average: 1.5627022653721951 (0.449%)
+Median: 2 (0.552%)
+Min: 2 (50.000%), max: -4 (-0.855%), delta: -6 (-1.293%)
+
```

Comment:
a plus sign for increasing delta would be also nice 
```suggestion
Min: +2 (+50.000%), max: -4 (-0.855%), delta: -6 (-1.293%)
```

Reply:
hmm... I'll just mention, I prefer positive values without the `plus` sign.  Treat it is like any other numeric literal (IMO 😄 )

Reply:
then let's keep it like it is - you will be using it more anyway probably 😃

Reply:
Left as is for now 😃